README.md

```diff
@@ -1,2 +1,62 @@
-# ecr-repository-lifecycle
-This repo contains a simple CloudFormation template for creating a lifecycle policy for images in an Amazon ECR repository
+# ECR Repository and LifeCycle Policy
+
+This repo contains a small CloudFormation template for creating a lifecycle policy for images in an Amazon ECR repository. There are a variety of lifecycle policies that may be written that use age of images, number of images or even image tag value to limit the crowding in an ECR repository.
```

Comment:
I'd leave out the word "small" here as it unnecessarily minimizes the work you've done.